### PR TITLE
Scriptures in Recital have a simple description

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: All Scripture in the Clockwork Slab's recital has a simple description.
/:cl:

terrifyingly overpowered

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Thank you

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -234,6 +236,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/judicial_visor //Judicial Visor: Creates a judicial visor.
+	descname = "Stun Blast Glasses"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Delayed Area Stun" or something like that might work better

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 2]
```diff
@@ -283,6 +287,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/sentinels_compromise //Sentinel's Compromise: Allows the invoker to select a nearby servant convert their brute and burn damage into half as much toxin damage.
+	descname = "Targeted Healing"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Including that it only heals brute and burn is probably desirable

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 3]
```diff
@@ -526,6 +539,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/clockwork_proselytizer //Clockwork Proselytizer: Creates a clockwork proselytizer.
+	descname = "Necessary, Converts Things"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Converts Things" might give the illusion that it's like Guvax. Perhaps "Necessary, Converts Objects"?

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 4]
```diff
@@ -671,6 +690,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 //////////////////
 
 /datum/clockwork_scripture/create_object/anima_fragment //Anima Fragment: Creates an empty anima fragment
+	descname = "Construct Shell"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Soul Vessel Shell"

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 5]
```diff
@@ -781,6 +804,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/targeted/justiciars_gavel //Justiciar's Gavel: Deals extreme but temporary brain damage to a target with the inputted name and knocks them out for a brief time.
+	descname = "Stun Target Name"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Stun Designated Target"

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 6]
```diff
@@ -819,6 +843,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/interdiction_lens //Interdiction Lens: Creates a powerful obelisk that can perform a variety of powerful sabotages every five minutes.
+	descname = "Disable Machinery Structure"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Structure, Disables Machinery"

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 7]
```diff
@@ -837,6 +862,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/mending_motor //Mending Motor: Creates a prism that will quickly heal mechanical servants/clockwork structures and consume replicant alloy.
+	descname = "Repair Structures Structure"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Structures, Repairs Other Structures"

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 8]
```diff
@@ -855,6 +881,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/clockwork_obelisk //Clockwork Obelisk: Creates a powerful obelisk that can be used to broadcast messages or open a gateway to any servant or clockwork obelisk.
+	descname = "Teleport Nexus Structure"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Structure, Teleportation Hub"

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 9]
```diff
@@ -873,6 +900,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/mania_motor //Mania Motor: Creates a powerful obelisk that can be used to broadcast messages or open a gateway to any servant or clockwork obelisk.
+	descname = "Area Denial Structure"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Structure, Area Denial"

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 10]
```diff
@@ -1040,6 +1072,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/invoke_inathneq //Invoke Inath-Neq, the Resonant Cogwheel: Grants a huge health boost to nearby servants that rapidly decreases to original levels.
+	descname = "Invulnerability"
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: "Area Invulnerability"

**Comment by Xhuis** [comment]
Kay, that should be all

File: code/game/gamemodes/clock_cult/clock_scripture.dm (hunk 10 above)
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Too large for the menu!

**Comment by InsaneHyena** [comment]
Fuck yeah. I mean, I've already figured out how to play Ratvar because I'm so great, but a lot of people can't seem to. They keep asking questions like "How to convert?" and "How to stun?" and I don't blame them, since the names for Ratvar magic are so strange. Compare "Invoke Inath-Neq, the Resonant Cogwheel" to, say, "Blood boil". So, this is a great and welcome change that must be merged as soon as possible.

**Pull request merged**